README.md

```diff
@@ -43,17 +44,50 @@ Usage: ruleguard [-flag] [package]
 
 Flags:
   -rules string
-    	path to a rules.go file
+        comma-separated list of gorule file paths or URLs
   -e string
-    	execute a single rule from a given string
+        execute a single rule from a given string
   -fix
-    	apply all suggested fixes
+        apply all suggested fixes
   -c int
-    	display offending line with this many lines of context (default -1)
+        display offending line with this many lines of context (default -1)
   -json
-    	emit JSON output
+        emit JSON output
 ```
 
+
+Create a test `example.go` target file:
```

Comment:
what was the point to change this part of the README? 👀

Reply:
It was not changed but rearranged. The text remains the same, git is bad at showing moved things. The motivation is to simplify the quick start.

**Before:**

+ Quick start
    + create example code
    + create example rules
    + run ruleguard using the created rules file

**Now:**

+ Quick start
    + create example code
    + run ruleguard using remote rules
+ Using custom rules
    + create example rules
    + run ruleguard using local rules